recirq/quantum_chess/quantum_board_test.py

```diff
@@ -78,7 +79,14 @@ def test_classical_jump_move(board):
     b.do_move(m)
     assert_samples_in(b, [u.squares_to_bitboard(['b1', 'c1'])])
 
-
+def test_path_qubits_error():
+    """ValueError should be returned when souce and target is not in the same line."""
+    b = qb.CirqBoard(u.squares_to_bitboard(['a1']))
+    with ShouldRaise(ValueError):
```

Comment:
You can use with pytest.raises and you do not need the new package.

Reply:
Thanks Doug it's changed.